### PR TITLE
fix "read more" button

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/kubernetes/website/issues/24775

### CODE REVIEW

**Comment by Arhell** [comment]
review page
https://deploy-preview-24826--kubernetes-io-master-staging.netlify.app/community/
/assign @sftim

File: static/css/newcommunity.css [hunk 1]
```diff
@@ -244,7 +244,7 @@ a {
   display: block;
   margin: auto;
   margin-top: 2rem;
-  width: 156px;
+  width: 164px;
```
Comment (zacharysarah) [inline static/css/newcommunity.css:247]: @Arhell 👋  At 164px, I still see a two-line button, both locally and in the Netlify preview:

Screen Shot 2020-11-16 at 11 59 16 AM

It looks like the minimum width for button text on a single line is 166px.

Screen Shot 2020-11-16 at 12 10 42 PM

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@Arhell 👋  Thanks for this fix. It's almost there! 

/cc @celestehorgan for visibility on CSS changes

File: static/css/newcommunity.css (hunk 1 above)
Comment (kbhawkey) [reply static/css/newcommunity.css:247]: Hi @Arhell . Viewing from a smallish laptop, I see the two lines of text when my browser reaches the maximum width of the screen. At smaller widths, the button text displays in a single line.
Is there a better way of setting the width of the button to adjust for the size of the container (screen size)?

Comment (Arhell) [reply static/css/newcommunity.css]: done
Made it independent of the width
@zacharysarah 
@kbhawkey 
Thanks!

**Comment by kbhawkey** [comment]
Change is isolated to community page.
/lgtm

**Pull request merged**